### PR TITLE
Feature/graph tabs

### PR DESCRIPTION
**Pull request by Petr26**

### CODE REVIEW

File: frontend/src/app/utils/format.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {formatCurrency} from '@angular/common';
+
+export function formatCZK(value: number, currencySymbol = 'Kč'): string {
+  return formatCurrency(value, 'cs-CZ', currencySymbol, 'symbol', '1.0-0');
```
Comment (jnwltr) [inline frontend/src/app/utils/format.ts:4]: I would prefer `.toLocalString()`, since it uses user's settings. And we already use it in the app.

Comment (jnwltr) [reply frontend/src/app/utils/format.ts:4]: Maybe, there's an issue with decimal points to solve. So just check it, pls.

File: frontend/src/app/game/game.service.ts [hunk 2]
```diff
@@ -26,6 +26,15 @@ export class GameService {
   private _infectedToday$ = new Subject<ChartValue>();
   infectedToday$ = this._infectedToday$.asObservable();
 
+  private _costTotal$ = new Subject<ChartValue>();
+  costTotal$ = this._costTotal$.asObservable();
+
+  private _deathToday$ = new Subject<ChartValue>();
+  deathToday$ = this._deathToday$.asObservable();
+
+  private _immunized$ = new Subject<{resistant: ChartValue; vaccinated: ChartValue}>();
```
Comment (jnwltr) [inline frontend/src/app/game/game.service.ts:35]: Having separate subjects for all numbers is not a good idea imo - lot of rigid code. Pls, merge the subjects to report the whole state, and subscribers just

**Review by jnwltr — CHANGES_REQUESTED** [review]
Pls, reflect my 2 minor comments.

File: frontend/src/app/game/game.service.ts (hunk 2 above)
Comment (Petr26) [reply frontend/src/app/game/game.service.ts:35]: I need to use it on multiple places and I dont want to deal with accessing exact property of the object. Also graph take as Input param `tick$` which is observable with data for that graph. 

And its not all the numbers, its now 3 and will be one more for the Immunity graph. 

I see it the opposite way, to have multiple observables with specific data.

File: frontend/src/app/utils/format.ts (hunk 1 above)
Comment (Petr26) [reply frontend/src/app/utils/format.ts:4]: Good point, changed. (Decimal point was not problem)

File: frontend/src/app/game/game.service.ts (hunk 2 above)
Comment (jnwltr) [reply frontend/src/app/game/game.service.ts:35]: I get your point. Yet we will consume other results/state props in other places, imagine the status panel.

The only thing I do not want is a necessity of adding extra and extra subjects in the game service for every consumer. If you want to mutate the state, just .map it locally. What do you think?

Comment (Petr26) [reply frontend/src/app/game/game.service.ts:35]: I get your point also, but I need to use it like here, its game.component.ts:
```
<mat-tab [label]="'Nově nakažení: ' + (gameService?.infectedToday$ | async)?.value || '0'">
          <cvd-chart [tick$]="gameService.infectedToday$"
                       [series]="[{label: 'Nově nakažení', unit: 'tis.'}]"></cvd-chart>
</mat-tab>
```

once with async map, one without - nice example how handy it is IMO.

But as always, I dont have problem to replace all those observables by one with whole state. Then I kinda just move the decomposition to another place (to game.component.ts). 

If that is ok, I will do that :)

**Review by jnwltr — APPROVED** [review]

**Pull request merged**